APCAppCore/APCAppCore/Library/Objects/APCTasksReminderManager.h

```diff
@@ -81,4 +82,22 @@ static NSUInteger const kAPCTaskReminderDayOfWeekSaturday    = 7;
  */
 + (NSString *)studyName;
 
+/*
+ * Right now there are a few conditions for determining if a task should be reminded about...
+ *
+ * 1) for any task, if the taskId was not in the old list, and is now in the new list
+ * 2) for activities, if the task was completed in old, but is incomplete in new
+ * 3) for activities, if the task is incomplete, old task is less than 3 days old, and new
+ *    task is more than 3 days old
+ *
+ * @param oldTaskGroups the result from reading cache BEFORE sycning from the web
+ * @param newTaskGroups the result from reading cache AFTER  sycning from the web
+ *
+ * @return an array of task group objects that are considered "new" per the 3 conditions above
+ */
++ (NSArray*)findNewTasksFromOld:(NSArray*)oldTaskGroups
+                toNewTaskGroups:(NSArray*)newTaskGroups;
+
++ (BOOL) isMoreThan3DaysOld:(APCTask*) task;
```

Comment:
Can it be more than X days old, with an age set in the app delegate? Other apps might need variable length.